tsl/test/expected/continuous_aggs.out

```diff
@@ -1738,3 +1738,107 @@ SELECT
 --
 (0 rows)
 
+-- test that option create_group_indexes is taken into account
+SET client_min_messages = ERROR;
+CREATE TABLE test_group_idx (
+time timestamptz,
+symbol int,
+value numeric
+);
+select create_hypertable('test_group_idx', 'time');
+      create_hypertable       
+------------------------------
+ (48,public,test_group_idx,t)
+(1 row)
+
+insert into test_group_idx
+(select generate_series(
+	'2020-01-01',
+    '2020-02-25', INTERVAL '30 sec'
+  ),
+  round(random()*10),
+  random()*5);
```

Comment:
```suggestion
insert into test_group_idx
select t, round(random()*10), random()*5
from generate_series('2020-01-01', '2020-02-25', INTERVAL '12 hours') t;
```
Why too many rows in the tests?? Do we really need it??? What about use `12 hours` so it will produce two rows for each time bucket.